app/uk/gov/hmrc/advancevaluationrulings/services/DmsSubmissionService.scala

```diff
@@ -59,8 +59,17 @@ class SaveFileDmsSubmissionService @Inject() (
 
   private def writeFile(pdfBytes: Array[Byte], submissionReference: String): Future[String] = {
     val fileName = "applications/" + "application.pdf"
-    Files.write(Paths.get(fileName), pdfBytes)
-    Future.successful(fileName)
+    try {
+
+      Files.write(Paths.get(fileName), pdfBytes)
+      Future.successful(fileName)
+    }
+    catch {
+      case _: IOException =>
+        logger.info("Failed to write local file")
+        Future.never
```

Comment:
make this e: IOException => and return a Future.failed(e). I'm cautious about what a Future.never does.

You could probably also raise it to an error or warning log rather than just info, I think that'd be reasonable if we left this in long-term.